601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/BookResource.java

```diff
@@ -51,4 +62,9 @@ public Long customFindPublicationIsbnObject(@PathParam("bid") Integer bid) {
         return bookRepository.findByPublisherAddressZipCode(zipCode);
     }
 
+    private void verifyFruitExists(Integer id) {
```

Comment:
```suggestion
    private void verifyBookExists(Integer id) {
```
Although I must admit that some books can be just as fresh and delicious. 😊